modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -549,27 +549,27 @@ There are several known issues in {productname} 6.2.
 
 === Merge Tags cannot be manually added to a {productname} document in some circumstances
 
-There are several circumstances where typing the Merge tags prefix (by default, `+{{+`) does not activate the `autocompleter`. Consequently, choosing a Merge tag from the expected pop-up menu is not possible because the pop-up menu does not present.
+There are several circumstances where typing the Merge Tags prefix (by default, `+{{+`) does not activate the `autocompleter`. Consequently, choosing a Merge tag from the expected pop-up menu is not possible because the pop-up menu does not present.
 
 Moreover, in the circumstances where the `autocompleter` does not activate, even typing the entire Merge tag does not result in the string being turned in to a Merge tag.
 
 There is a workaround, however.
 
-If typing the Merge tags prefix does not work as expected, clicking the Merge tags toolbar button and selecting the desired Merge tag from the resultant menu inserts a functioning Merge tag.
+If typing the Merge Tags prefix does not work as expected, clicking the Merge Tags toolbar button and selecting the desired merge tag from the resultant menu inserts a functioning merge tag.
 
-=== Merge tags does not validate inserted strings that match merge tag syntax
+=== Merge Tags does not validate inserted strings that match merge tag syntax
 
 If a string is pasted or otherwise inserted into a {productname} document and that string presents as a complete merge tag, the Merge tag plugin will turn the string into an apparently working merge tag, regardless of the string’s content.
```

Comment:
```suggestion
If a string is pasted or otherwise inserted into a {productname} document and that string presents as a complete merge tag, the Merge Tags plugin will turn the string into an apparently working merge tag, regardless of the string’s content.
```